src/components/m-footer-links/js/footer-links.js

```diff
@@ -31,18 +33,22 @@ export default class FooterLinks {
   on() {
     this.off();
 
-    this.unsubscribe = subscribe('device-state/change', (event) => {
-      const { detail: { breakpoint } } = event;
+    const { wcNode } = this;
+
+    this.unsubscribe = this.deviceStateObserver.listen((state) => {
+      const { breakpoint } = state;
       const hasDropdown = hasDropdownBreakpoints.indexOf(breakpoint) > -1;
 
       if (hasDropdown && !this.dropDown) {
-        this.dropDown = new DropDown(this.wcNode);
+        this.dropDown = new DropDown(wcNode);
       } else if (!hasDropdown && this.dropDown) {
         this.dropDown.destroy();
         delete this.dropDown;
       }
     });
 
+    this.deviceStateObserver.triggerOnce();
```

Comment:
shouldn't that come directly form the `deviceStateObserver `?

Reply:
what if you dont want the initial callback and only onchange?